style.css

```diff
@@ -12,6 +12,37 @@ html {
     text-align: center; 
 }
 
+#header-bg {
+    background-color: var(--background-color-header);
+    height: 80px;
+}
+
+#header {
+    margin: 0em 5em;
+    min-width: auto;
+    max-height: auto;
```

Comment:
we should make a nav bar fixed at the top so when users scrolling down our page the nav bar would be still there. That said, you need to set position as fixed and set a height for it